userProfile/__tests__/end_to_end_unit.test.mjs

```diff
@@ -0,0 +1,53 @@
+import userProfileRouter from '../index.mjs'
+import express from 'express'
+import request from 'supertest'
+
+const routeTester = new express()
+routeTester.use("/user", userProfileRouter)
+
+describe('userProfile endpoint end to end unit test (spinning up the endpoint and using it). #end2end_unit', () => {
+
+  it('POST instead of GET. That status should be 501 with a message.', async () => {
+    const res = await request(routeTester)
+      .post('/user/')
+    expect(res.statusCode).toBe(501)
+    expect(res.body).toBeTruthy()
+  })
+
+  it('PUT instead of GET. That status should be 501 with a message.', async () => {
+    const res = await request(routeTester)
+      .put('/user/')
+    expect(res.statusCode).toBe(501)
+    expect(res.body).toBeTruthy()
+  })
+
+  it('PATCH instead of GET. That status should be 405 with a message.', async () => {
+    const res = await request(routeTester)
+      .patch('/user/')
+    expect(res.statusCode).toBe(405)
+    expect(res.body).toBeTruthy()
+  })
+
+ 
```

Comment:
Why the extra newline whitespace?  If it doesn't serve a purpose just reduce it to a single newline.

Reply:
removed, updated.